src/ralph/backends/data/swift.py

```diff
@@ -51,16 +51,18 @@ class Config(BaseSettingsConfig):
         env_prefix = "RALPH_BACKENDS__DATA__SWIFT__"
 
     AUTH_URL: str = "https://auth.cloud.ovh.net/"
-    USERNAME: Optional[str] = None
-    PASSWORD: Optional[str] = None
+    DEFAULT_CONTAINER: Optional[str] = None
     IDENTITY_API_VERSION: str = "3"
-    TENANT_ID: Optional[str] = None
-    TENANT_NAME: Optional[str] = None
+    OBJECT_STORAGE_URL: Optional[str] = None
+    PASSWORD: Optional[str] = None
```

Comment:
I don't think these changes are supposed to be here?

Reply:
Good point) Thanks) Moved it to a separate commit)